package.json

```diff
@@ -13,16 +13,20 @@
     "prepublishOnly": "yarn build && yarn copy-files && cp -r ./build/* .  && rm -rf ./build",
     "revert:bundle": "git clean -fd"
   },
-  "engines": {
```

Comment:
Which was the reason for delete engines ?

Reply:
What do you mean?